Containerfile

```diff
@@ -0,0 +1,20 @@
+FROM debian
+
+LABEL org.opencontainers.image.source https://github.com/hulks/meta-nao
+
+RUN apt-get update && apt-get install --no-install-recommends --yes \
+  ca-certificates \
+  curl \
+  file \
+  python3 \
+  xz-utils \
+  zstd \
+  && rm --recursive --force /var/lib/apt/lists/*
+
+ARG version
+
+RUN curl --fail --location --progress-bar --output /opt/toolchain.sh \
+  https://github.com/HULKs/meta-nao/releases/download/${version}/HULKs-OS-$(uname -m)-toolchain-${version}.sh
```

Comment:
I'd expect the SDK to come from the build host. Not downloaded from releases. This allows this `Containerfile` to be reused even for builds that are not released.

Reply:
Good point, fixed.